concepts/05 Widgets/Scheduler/040 Resources/030 Group Appointments by Resources.md

```diff
@@ -108,7 +190,7 @@ You can change resource headers' orientation in an individual view using the **v
     })
 
     <!--HTML-->
-    <dx-scheduler ... 
+    <dx-scheduler ...
         [views]="views">
     </dx-scheduler>
```

Comment:
This example misses Vue and React versions.

Reply:
Done